api/go.mod

```diff
@@ -19,3 +19,5 @@ require (
 	sigs.k8s.io/kustomize/kyaml v0.4.1
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace sigs.k8s.io/kustomize/kyaml v0.4.1 => ../kyaml
```

Comment:
Why do we add a replace for kyaml here? I don't see any change in kyaml for this PR.

Reply:
it's just for continued development.
I could do it in a different PR if you prefer, just being lazy